networking/enable-cluster-wide-proxy.adoc

```diff
@@ -34,3 +34,4 @@ include::modules/nw-proxy-remove.adoc[leveloffset=+1]
 
 * xref:../security/certificates/updating-ca-bundle.adoc#ca-bundle-understanding_updating-ca-bundle[Replacing the CA Bundle certificate]
 * xref:../security/certificate_types_descriptions/proxy-certificates.adoc#customization[Proxy certificate customization]
+* link:https://access.redhat.com/solutions/7065528[How is the cluster-wide proxy setting applied to OCP nodes?]
```

Comment:
I would +1 this bot comment, even tho this is the technically the actual title of the target KBase article.
```suggestion
* link:https://access.redhat.com/solutions/7065528[How is the cluster-wide proxy setting applied to {product-title} nodes?]
```
Might as well spell it out since we don't want "OCP" really in the docs. You (or the KBase author) could also update the KBase title to match, if you'd like.

Reply:
Thanks! I wasn't sure how to handle that since the article's title says "OCP" but I'll see about having the article updated.

Reply:
... and I'll update the the link (as you suggested) :)